api/web/base.py

```diff
@@ -208,6 +210,35 @@ def validate_oauth_token(self, access_token, timestamp):
 
         return uid
 
+
+    @log_access(AccessType.user_login)
+    def log_in(self):
+        """
+        Return succcess boolean if user successfully authenticates.
+
+        Used for access logging.
+        Not required to use system as logged in user.
+        """
+
+        if not self.uid:
+            self.abort(400, 'Only users may log in.')
+
+        return {'success': True}
```

Comment:
Depending on policy this could be changed to log whenever a user has an auth_token (eventually session token) inserted into the db but for now it's just a 1:1 for a presumed button on a frontend client.